contracts/LimitOrderSwap.sol

```diff
@@ -195,7 +198,11 @@ contract LimitOrderSwap is ILimitOrderSwap, Ownable, TokenCollector, EIP712, Ree
         uint256 orderFilledAmount;
         (orderHash, orderFilledAmount) = _validateOrder(_order, _makerSignature);
 
+        if (_takerTokenAmount == 0) revert ZeroTakerSpendingAmount();
+        if (_makerTokenAmount == 0) revert ZeroMakerSpendingAmount();
+
         // get the quote of the fill
+        // orderAvailableAmount must be greater than 0 here, or it will be reverted by the _validateOrder function
```

Comment:
Is this comment for coverage?

Reply:
Nope, it just reminds developers or auditors of the implicit condition.

Reply:
But why do you need a comment for it? You are not using `unchecked` to do the arithmetics.

Reply:
It just helps me clarify what `_makerTokenAmount` could be if `orderAvailableAmount` is always greater than 0. However, we could actually use an `unchecked` block to wrap the arithmetic operations."

Reply:
If it's for `unchecked` then no problem. Otherwise it might not help much as reviewers still have to go over the code to check the conditions themselves.